MediaAdminBundle/Resources/public/coffee/bindExtendView.coffee

```diff
@@ -0,0 +1,17 @@
+callback_tinymce_init = null
+widgetChannel.bind 'ready', (view) ->
+  isMediaView = $("[data-prototype*='clear-media']", view.$el).length > 0
+  isWysiwygView = $(".select_media_modal", view.$el).length > 0
+  if isMediaView or isWysiwygView
+    $.extend true, view, extendView['orchestraMediaAbstractType']
+    if isMediaView
+      $.extend true, view, extendView['orchestraMediaType']
+    if isWysiwygView
+      do (view) ->
+        callback_tinymce_init = (editor) ->
+          $.extend true, view, extendView['orchestraWysiwygType']
```

Comment:
you should send view.delegatesEvents here in order to bind in backbone view.
In this way, you could bin in extend view on [aria-label="mediamanager"] and not use on('click')